contrib/IECoreArnold/src/IECoreArnold/RendererImplementation.cpp

```diff
@@ -542,24 +547,32 @@ void IECoreArnold::RendererImplementation::illuminate( const std::string &lightH
 
 void IECoreArnold::RendererImplementation::motionBegin( const std::set<float> &times )
 {
-	if( !m_motionTimes.empty() )
+	if( m_motionBlockSize )
 	{
 		msg( Msg::Error, "IECoreArnold::RendererImplementation::motionBegin", "Already in a motion block." );
 		return;
 	}
 
-	m_motionTimes.insert( m_motionTimes.end(), times.begin(), times.end() );
+	// TODO - is it worth the time to do this, in order to avoid silently giving incorrect results?
```

Comment:
Seems unlikely to have significant overhead compared to the rest of the work we're doing.